src/generic_core/core.ts

```diff
@@ -279,7 +280,8 @@ class uProxyCore implements uProxy.CoreAPI {
    * instances.
    */
 
-  public updateGlobalSettings = (newSettings:Core.GlobalSettings) => {
+  public updateGlobalSettings = (newSettings :Core.GlobalSettings) => {
+    newSettings.version = uProxy.STORAGE_VERSION;
```

Comment:
No, they might be different.  For example, the globalSettings could have been previously stored with version=1.  Then after uProxy updates to version 2, we load globalSettings again with version=1, and at the time updateGlobalSettings was called globalSettings.version would not be the same as uProxy.STORAGE_VERSION.

Also we don't want to throw any errors yet.  We are just putting these fields in as place holders so we can rely on them being available later if we make backwards incompatible changes